src/Raven.Client/Documents/Smuggler/DatabaseItemType.cs

```diff
@@ -11,6 +11,6 @@ public enum DatabaseItemType
         RevisionDocuments = 1 << 1,
         Indexes = 1 << 2,
         Transformers = 1 << 3,
-        Identities = 1 << 4,
+        ClusterIdentities = 1 << 4
```

Comment:
@myarichuk please open an issue that we need to support this block in 3.x (by skipping it). IIRC the 3.x smuggler throws if not supported property is encountered.